Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -59,19 +59,24 @@ def edit
           "#{path} doesn't exist on disk."
         end
 
-        message = if args.cask?
-          <<~EOS
-            #{not_exist_message}
-            Run #{Formatter.identifier("brew create --cask --set-name #{path.basename(".rb")} $URL")} \
-            to create a new cask!
-          EOS
+        suggestion_message = if args.cask?
+          if Tap.fetch("homebrew", "cask").path.exist?
+            "Run #{Formatter.identifier("brew create --cask --set-name #{path.basename(".rb")} $URL")} " \
+              "to create a new cask!"
+          else
+            "Run #{Formatter.identifier("brew tap homebrew/cask")} to tap homebrew/cask!"
+          end
+        elsif Tap.fetch("homebrew", "core").path.exist?
+          "Run #{Formatter.identifier("brew create --set-name #{path.basename} $URL")} " \
+            "to create a new formula!"
         else
-          <<~EOS
-            #{not_exist_message}
-            Run #{Formatter.identifier("brew create --set-name #{path.basename} $URL")} \
-            to create a new formula!
-          EOS
+          "Run #{Formatter.identifier("brew tap homebrew/core")} to tap homebrew/core!"
```

Comment:
Would it be possible to check to see if the formula/cask is actually in the core repo before suggesting tapping it? Also, wouldn't it need to be `brew tap --force homebrew/core` in this case?

Reply:
I'm worried about the case where we tell someone to tap the core repo and then the formula/cask isn't even available there.

Reply:
Yeh, could use the API JSON for that for sure.

Reply:
So the question then is: if the core tap is not present and the requested path is not present, do we 1) say to run `brew tap` and `brew create` in one go, or, 2) just say to run `brew create` and then adjust `dev-cmd/create.rb` to suggest running `brew tap` instead of just saying "No available tap homebrew/core"?

Reply:
I've went with 2).